RecoLocalTracker/SiPixelClusterizer/plugins/GPUClusterThresholds.h

```diff
@@ -0,0 +1,9 @@
+#ifndef RecoLocalTracker_SiPixelClusterizer_plugins_GPUClusterThresholds_h
+#define RecoLocalTracker_SiPixelClusterizer_plugins_GPUClusterThresholds_h
+
+struct GPUClusterThresholds {
```

Comment:
Since we are at it we can remove `GPU` completely form the name.
Those constant have nothing to do with GPUs, and in fact are the same in the legacy code configuration:

https://github.com/cms-sw/cmssw/blob/9c0ef283f7847d7aca1e34791522637609dcee87/RecoLocalTracker/SiPixelClusterizer/python/SiPixelClusterizer_cfi.py#L15-L16

BTW, @slava77 if 

> some input on naming of the parameter structs from the DPG may be useful, especially if this ever makes it to a DB.

you were referring to this, I don't think we're planning to store this in DB. Once the new L1 is timed again and we know what threshold optimizes resolution we don't plan to change then again (TM)

Reply:
done, and without the GPU suffix